MongoConsumerLibary/MongoConnection/Collections/BaseBoxCollection.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace MongoConsumerLibary.MongoConnection.Collections
+{
+    public class BaseBoxCollection
```

Comment:
Add two params for this class: Dictionary of all params with decoded params and IcdType